packages/stockflux-launcher/src/App.css

```diff
@@ -1,38 +1,64 @@
 html {
-  background-color: var(--middle-background);
+  background-color: var(--launcher-color);
```

Comment:
Is this color different from the other apps? If not, i would suggest just using `--window-background-color` or at leat specifying 
```
:root {
  --launcher-color: var(--window-background-color)
}
```
That way, if we ever decide to change `--window-background-color` we can do it all in one place. 

I've noticed there are some more hardcoded colors in `:root`, e.g.`--titlebar-color`, `--search-input-text-color: #ffffff`. I think that is used throughout the apps and again, should not be declared/redeclared in the launcher app specifically. It's more global than the launcher. See `packages\stockflux-components\src\index.css`.